spectral_rescaling.py

```diff
@@ -0,0 +1,74 @@
+import torch
+import torch.nn.functional as F
+
+
+def compute_spectral_rescaling_dense(M, n_iter=1):
+    n_iter = max(n_iter, 1)
```

Comment:
Avoid silent input correction.
```suggestion
    if n_iter < 1:
        raise ValueError(f"n_iter must be at least equal to 1, got {n_iter}")
```